.storybook/main.js

```diff
@@ -19,7 +19,15 @@ module.exports = {
     "@storybook/addon-a11y"
   ],
   typescript: {
-    reactDocgen: false
+    check: false,
+    checkOptions: {},
+    // ideally we use `react-docgen-typescript`. But there's still some issue, related to webpack 5.
+    // so we use `react-docgen` for now
+    reactDocgen: 'react-docgen', 
+    reactDocgenTypescriptOptions: {
+      shouldExtractLiteralValuesFromEnum: true,
+      propFilter: (prop) => (prop.parent ? !/node_modules/.test(prop.parent.fileName) : true),
+    },
```

Comment:
Why are we doing this?

Reply:
![image](https://user-images.githubusercontent.com/12745166/133369996-d44c6f05-b322-4bed-a5db-ebda238a6883.png)

This `react-docgen` infers some information from the source code. Such as `default` value in the table.